src/hooks/useCds/translate.js

```diff
@@ -259,6 +259,15 @@ const loincBiopsyReport = {
 const immunosuppressedObservation = {
   label: 'Immunosuppressed',
   code: {
+    coding: [
+      {
+        system: LOINC_URL,
+        code: 96381-9,
```

Comment:
Does new immunosuppressed LOINC code need single quotes, like the other codes we define here?

Reply:
Upon testing locally with https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/136 as a dependency, the wrong code is passed onto the patient as-is, and toggling the immunocompromised switch does not cause a change in patient data. 

When this code is placed in single quotes, the correct code is passed onto the patient, and you can see one of the Logica Health patients getting a screening recommendation from the Immunocompromised logic path as opposed to the average risk logic path.